api/src/test/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceTest.java

```diff
@@ -619,6 +619,31 @@ public void testUseAbsorb_FalseWhenMoodlePreviouslyUsed() throws Exception {
     assertThat(complianceTrainingService.useAbsorb()).isFalse();
   }
 
+  @Test
+  public void testTrainingsEnabled_TrueWhenMigrationPauseInactive() throws ApiException {
+    setRTTrainingCompletedWithMoodle();
+    providedWorkbenchConfig.moodle.trainingMigrationPauseActive = false;
+
+    assertThat(complianceTrainingService.trainingsEnabled()).isTrue();
+  }
+
+  @Test
+  public void testTrainingsEnabled_TrueWhenMoodleNeverUsed() {
+    providedWorkbenchConfig.moodle.trainingMigrationPauseActive = true;
+    // setRTTrainingCompletedWithMoodle is not called
+
+    assertThat(complianceTrainingService.trainingsEnabled()).isTrue();
+  }
+
+  @Test
+  public void testTrainingsEnabled_FalseWhenMigrationPauseActiveAndUsedMoodle()
```

Comment:
Should we have another set of test for providedWorkbenchConfig.moodle.trainingMigrationPauseActive is TRUE and user is due for Absorb

Reply:
I think that's reasonable. I added an Absorb test.